assets/dev/js/frontend/handlers/section/section.js

```diff
@@ -1,13 +1,18 @@
 import BackgroundSlideshow from '../background-slideshow';
 import BackgroundVideo from '../background-video';
-import HandlesPosition from './handles-position';
+import HandlesPosition from '../handles-position';
 import StretchedSection from './stretched-section';
 import Shapes from './shapes';
 
-export default [
+const handlers = [
 	StretchedSection, // Must run before BackgroundSlideshow to init the slideshow only after the stretch.
 	BackgroundSlideshow,
 	BackgroundVideo,
-	HandlesPosition,
 	Shapes,
 ];
+
+if ( elementorFrontend.isEditMode() ) {
+	handlers.push( HandlesPosition );
+}
```

Comment:
aren't the section handlers dynamically loaded? ewwww


(not your fault, be we _have_ to fix that. it probably loads these scripts even when you don't have sections in the page)